bin/release-changelog/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+const Changelog = require( './changelog' );
+
+const milestone = process.argv[ 2 ];
+
+const token = process.env.GITHUB_TOKEN;
```

Comment:
If `token` is empty, should this raise an error to inform the user to add it to their environment?

Reply:
Yea that makes sense. I've added some logic to properly parse the args and validate them in 312f0d389a5e1de007ebf45fbb83090f017e4296.